processing/src/main/java/io/druid/query/search/search/ContainsSearchQuerySpec.java

```diff
@@ -0,0 +1,102 @@
+package io.druid.query.search.search;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.metamx.common.StringUtils;
+
+import java.nio.ByteBuffer;
+
+/**
+ */
+public class ContainsSearchQuerySpec implements SearchQuerySpec
+{
+
+  private static final byte CACHE_TYPE_ID = 0x1;
+
+  private final String value;
+  private final boolean caseSensitive;
+
+  private final String target;
+
+  @JsonCreator
+  public ContainsSearchQuerySpec(
+      @JsonProperty("value") String value,
```

Comment:
I don't know. Some methods are checking null and some are not. I'll update to handle nulls in all methods.